shopinvader_api_delivery_carrier/models/shopinvader_backend.py

```diff
@@ -9,6 +9,7 @@ class ShopinvaderBackend(models.Model):
 
     _inherit = "shopinvader.backend"
 
+    # TODO: what to do with it?
     delivery_order_states = fields.Selection(
```

Comment:
This selection field configured, on the shopinvader.backend, which picking should be visible from the front, depending on their states.
What to do with this configuration now since we have no more shopinvader.backend?

Reply:
We don't keep this feature here because we don't need it. It may be added later on.